markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -69,6 +73,28 @@ library PerpsAccount {
         }
     }
 
+    function validateMaxPositions(Data storage self, uint128 marketId) internal view {
+        if (PerpsMarket.accountPosition(marketId, self.id).size == 0) {
+            uint128 maxPositionsPerAccount = GlobalPerpsMarketConfiguration
+                .load()
+                .maxPositionsPerAccount;
+            if (maxPositionsPerAccount <= self.openPositionMarketIds.length()) {
```

Comment:
why <= instead of just <?

if max is set to 5, can't you have 5 open positions?

Reply:
because we check here if we can add another => in your example we'll check when adding the 5th so we need to have 4 already there at most